src/comment.ts

```diff
@@ -59,7 +59,7 @@ export function renderMarkdown({
   // Build footer
   let footer = ''
   if (includeFooter === undefined || includeFooter === true) {
-    footer = `\n\n---\n\n_Triggered by @${github.context.actor}`
+    footer = `\n\n---\n\nTriggered by @${github.context.actor}`
```

Comment:
Ah nice catch :smile: I think we just forgot the closing character for making it italic :)

```suggestion
    footer = `\n\n---\n\n_Triggered by @${github.context.actor}_`
```